client/extensions/woocommerce/state/sites/shipping-zones/test/selectors.js

```diff
@@ -77,12 +127,20 @@ describe( 'selectors', () => {
 			expect( areShippingZonesLoading( loadedEmptyState, 123 ) ).to.be.false;
 		} );
 
-		it( 'when zones are loaded but some methods are not.', () => {
-			expect( areShippingZonesLoading( loadingMethodsState, 123 ) ).to.be.true;
+		it( 'when zones are loaded but some methods and locations are not.', () => {
```

Comment:
Ouch, that's right. I fixed this file in another PR, I'll eventually need to merge / rebase and I'll change the new test cases then.